06-security.Rmd

```diff
@@ -96,7 +96,27 @@ In summary, be sure to separate "trusted" from "untrusted" objects in your API i
 
 ## Cross-Origin Resource Sharing (CORS)
 
-// TODO CORS
+Cross-origin resource sharing (CORS) is a mechanism that allows restricted resources (e.g. fonts) on a web page to be requested from another domain outside the domain from which the first resource was served. A web page may freely embed cross-origin images, stylesheets, scripts, iframes, and videos. Certain "cross-domain" requests, notably Ajax requests, are forbidden by default by the same-origin security policy.
+
+By default, the API endpoint forbids "cross-domain" requests, but it's possible with a workaround to override this behavior creating a custom function that'll forwards the right headers to API using a filter such as the following snippet.
+
+``` R
```

Comment:
I think we'd need

\```r

to get the syntax highlighting

Reply:
I think that both `r` and `R` will get the same result. I'm ok for `r` by the way.